test/e2e/__snapshots__/ClientMode.test.js.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`clientMode custom client on browser client logs additional messages to console 1`] = `
+Array [
+  "Hey.",
+  "open",
+  "liveReload",
+  "[WDS] Live Reloading enabled.",
+  "hash",
+  "ok",
```

Comment:
Why isn't `onClose` called?

Reply:
@hiroppy `testServer.close` happens after the snapshot is tested, so the client is not closed before that. I could maybe rearrange that to test `onClose` as well.